articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+
+<figure><img src="./images/logo-texture-regions.png" alt="Figure 4-12: The MonoGame logo broken down into texture regions."><figcaption><p><strong>Figure 4-12: The MonoGame logo broken down into texture regions.</strong></p></figcaption></figure>
+
+We can see from this image that the actual logo starts at position (0, 0) and is 128px wide and 128px tall. Likewise, the MonoGame word mark starts at position (150, 34) and is 458px wide and 58px tall. Knowing the starting position and the width and height of the region gives us a defined rectangle that we can use as the *sourceRectangle*.
+
+Let's see this in action by only drawing the MonoGame logo icon part of the texture. First, after the call to the [Clear](xref:Microsoft.Xna.Framework.Graphics.GraphicsDevice.Clear(Microsoft.Xna.Framework.Color)) method, add the following variable:
+
+```cs
+Rectangle iconSourceRect = new Rectangle(0, 0, 128, 128);
+```
+
+Next, update the `_spriteBatch.Draw` method call to the following
+
+```cs
+_spriteBatch.Draw(
+    _logo,              // texture
+    new Vector2(        // position
+      Window.ClientBounds.Width, 
+      Window.ClientBounds.Height) * 0.5f,
+    iconSourceRect,     // sourceRectangle
+    Color.White,        // color
+    0.0f,               // rotation
+    new Vector2(        // origin
+      iconSourceRect.Width, 
+      iconSourceRect.Height) * 0.5f,
+    1.0f,               // scale
+    SpriteEffects.None, // effects
+    0.0f);              // layerDepth
+```
+
+The changes you just made first added a new [Rectangle](xref:Microsoft.Xna.Framework.Rectangle) value called `iconSourceRect` that represents the dimensions of the MonoGame logo icon region within the texture. Then the *sourceRectangle* parameter of the `_spriteBatch.Draw` was updated to use the new `iconSourceRect` value. Notice that we are still telling it to draw the `_logo` for the *texture*, we've just supplied it with a source rectangle this time. Finally, the *origin* parameter was updated to use the width and height of the `iconSourceRect`. Since the overall dimensions of what we'll be rendering has changed due to supplying a source rectangle, the origin needs to be adjusted to those dimensions as well.
+
+If you run the game now, you should see the following:
+
+<figure><img src="./images/logo-only-centered.png" alt="Figure 4-13: The MonoGame logo icon only, from the logo texture, centered in the game window"><figcaption><p><strong>Figure 4-13: The MonoGame logo icon only, from the logo texture, centered in the game window</strong></p></figcaption></figure>
+
+As an exercise for yourself, see if you can adjust the code to draw only the MonoGame word mark from the logo texture the same way we just did the MonoGame logo icon.
+
+## Conclusion
+
+Let's review what you accomplished in this chapter:
+
+- You added an image file to the content project using the MGCB Editor

```

Comment:
We did also learn about loading from file.  However, the tutorial is missing an example of actually doing it (it only talks about it)?  Should it be included as a comparison?

Reply:
Added the following bullet point

> - You learned the disadvantages of loading an image file directly compared to the advantages of use the **Content Pipeline**